lighthouse-core/audits/audit.js

```diff
@@ -125,7 +125,12 @@ class Audit {
     if (displayValue === score) {
       displayValue = '';
     }
-
+    let auditDescription = audit.meta.description;
+    if (audit.meta.fallbackDescription) {
+      if (!result.rawValue || (typeof result.rawValue === 'number' && result.rawValue < 100)) {
```

Comment:
Maybe I missed this in the last PR review, but shouldn't this be based on `score`?